tasks/utils/logger.js

```diff
@@ -0,0 +1,26 @@
+import winston from 'winston';
+import expressWinston from 'express-winston';
+
+const loggerOptions = {
+    transports: [
+        new winston.transports.Console()
+    ],
+    format: winston.format.json(),
+    meta: false,
+    msg: 'HTTP {{req.method}} {{req.url}} {{res.responseTime}}ms',
+    expressFormat: false,
+    colorize: false
+};
+export const logger = expressWinston.logger(loggerOptions);
```

Comment:
This is a personal preference, however consider the possibility of grouping all exports at the bottom of the file in a barrel export.